File: maxElement.cpp
// Find the max element from the given array

#include<iostream>
using namespace std;
int main(){
    int n;
    cin>>n;
    int ar[n];
    for(int i=0;i<n;i++){
        cin>>ar[i];
    }
    int maxElement=0;
    for(int i=0;i<n;i++){
        maxElement=max(maxElement,ar[i]);
        cout<<maxElement<<" ";
    }
    return 0;
}

File: Max_element.java
Program to find the maximum element in an array

import java.util.*;
class HelloWorld {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        int ar[]= new int[n];
        for(int i=0;i<n;i++){
            ar[i]=sc.nextInt();
        }
        int m=ar[0];
        for(int i=0;i<n;i++){
            m=Math.max(m,ar[i]);
        }
        System.out.println(m);
    }
}
